app/src/main/java/fr/free/nrw/commons/db/Converters.java

```diff
@@ -0,0 +1,74 @@
+package fr.free.nrw.commons.db;
+
+import android.net.Uri;
+
+import androidx.room.TypeConverter;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+
+import fr.free.nrw.commons.location.LatLng;
+
+public class Converters {
+    private static Gson gson;
+    public static Gson getGson(){
+        if(null==gson){
+            gson=new Gson();
```

Comment:
Can we retrieve our actual gson instance? I don't care if it has to be exposed on the component level

Reply:
You mean a static instance from there?

Reply:
You can expose the gson instance by writing a method in the component
```
Gson gson();
```
calling this method will use the rules defined in the module to give us our Gson which is a singelton, you just need to get a handle on the Component

Reply:
Used the Global Gson object, provided by the NetworkModule